README.md

```diff
@@ -1,2 +1,54 @@
 # front-sdk
-An SDK for working with Front
+An SDK for working with Front (https://frontapp.com).
+
+## TypeScript Quickstart
+
+Import in the usual way:
+
+```typescript
+import { Front } from 'front-sdk';
+```
+
+## JavaScript
+
+Require in the usual way:
+
+```javascript
+const Front = require('front-sdk').Front;
+```
+
+## Usage
+
+Create a new instance of the SDK to use it, passing the API token for Front.
+
+```typescript
+import { Front, FrontComment } from 'front-sdk';
+const front = new Front('abcdef123457890');
+
+front.comment.create({
+    conversation_id: 'cnv_xxxx',
+    author_id: `alt:email:back@tofront`,
+    body: 'It is very important that you see this comment!'
+}).then((comment: FrontComment) => {
+    console.log(comment);
+}).catch((err: FrontError) => {
```

Comment:
You could add a
```typescript
.catch(FrontError, (err: FrontError) => {

})
```
which will actually check if it's a FrontError (and maybe if we're super lucky the typedef for bluebird will also automatically type enforce the function signature)